src/v2/ion/RemediationConstants.js

```diff
@@ -42,6 +42,68 @@ const FORMS = {
   TERMINAL: 'terminal',
 };
 
+const FORMNAME_CLASSNAME_MAPPING = {
+  'identify': ['identify', 'primary-auth'],
+  'identify-password': ['identify', 'primary-auth'],
+  'select-identify': ['select-identify'],
+  'enroll-profile': ['enroll-profile', 'registration'],
+
+  // 1st step of the challenge flow for sms/call
+  'authenticator-verification-data-sms': ['authenticator-verification-data-sms'],
+  'authenticator-verification-data-phone': ['authenticator-verification-data-phone'],
```

Comment:
feels the redundancy is not necessary since it can be created on the flay. (the `key` in the `getClassNameMapping` function)

Reply:
right, I think this can be removed.